ceph-lttng/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM ceph-heads/remotes/origin/mds-reqlatency
```

Comment:
The build script uses `dmake` to build an image from the remotes/origin/mds-reqlatency branch. After `dmake`, this Dockerfile builds from that image, which is called ceph-heads/remotes/origin/mds-reqlatency.